src/virtual-wan/azext_vwan/_help.py

```diff
@@ -478,6 +507,9 @@
 helps['network p2s-vpn-gateway update'] = """
     type: command
     short-summary: Update settings of a point-to-site VPN gateway.
+      - name: Update settings of a point-to-site VPN gateway with routing configuration.
```

Comment:
Should add a key `examples` before `-name` which failed CI.

Reply:
done